recipes/photospline/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: photospline
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/cnweaver/photospline/archive/v{{ version }}.tar.gz
+  sha256: 7045a631c41489085037b05fac98fd9cad73dc4262b7eead143d09e5f8265dec
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - numpy >=1.7
+    - suitesparse
+    - cfitsio
+    - cmake >=3.1
+    - {{ compiler('c') }}  # [not osx]
+    - {{ compiler('cxx') }}  # [not osx]
```

Comment:
Only these two should be in `build` the rest should probably be in `host`.
See https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run for more info.